kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -186,7 +186,6 @@ class KyuubiSessionImpl(
   override protected def runOperation(operation: Operation): OperationHandle = {
     if (operation != launchEngineOp) {
       waitForEngineLaunched()
-      sessionEvent.totalOperations += 1
```

Comment:
should we treat launchEngineOp as an operation?
cc @wForget

Reply:
> should we treat launchEngineOp as an operation? cc @wForget

This means whether we should treat `openSession` as an operation, i tend not to treat as an operation.